### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by TheShark823**
Creating the login page with a text field for email and another text field for password. this is only the GUI part first, no data validation is done yet.

### CODE REVIEW

**Comment by Bwise1** [comment]
@21907388 take note of this few things in next commit 

1. your commit message should be explanatory, for example "Created the email and password field for login page"
2. your pull request title and description also... you can add images
3. I will accept this for first time commits.

and also there seems to be an issue with the pages stuff. I will check the issues and let you know

File: src/app/login.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
+import React, { useState } from "react";
+import { NextPage } from "next";
+import styles from "./page.module.css";
+
+const LoginPage: NextPage = () => {
```
Comment (Bwise1) [inline src/app/login.tsx]: why are you using Nextpage?
the error comes from there.

you can make the page a normal function like this
`const LoginPage = () => {`
this should fix the error

File: src/app/login/page.tsx [hunk 2]
```diff
@@ -0,0 +1,92 @@
+"use client";
```
Comment (Bwise1) [inline src/app/login/page.tsx:1]: Why are there 2 login pages.
delete the one that is not active and leave the one.

**Review by Bwise1 — CHANGES_REQUESTED** [review]
@21907388 try and fix this changes and make another pull request thank you.

**Comment by TheShark823** [comment]
Capture

**Pull request merged**